### PR TITLE
chore: make it easier to run dev mode

### PR DESCRIPTION
**Pull request by aaronmgdr**
its hard to remember to run make dev 

`yarn dev` is more common

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "packages/*"
   ],
   "scripts": {
+    "dev": "make dev",
```
Comment (crondinini) [inline package.json:8]: Any reason why we wouldn't directly stop using Makefile and set up whatever `make dev` does here?
 i.e.:
```suggestion
    "dev": "lerna run dev --stream --parallel",
```

**Review by crondinini — APPROVED** [review]
I don't know the reason behind using Makefile, but if we need to keep it then this workaround is simple enough. 👍

File: package.json (hunk 1 above)
Comment (aaronmgdr) [reply package.json:8]: To be honest I didn't even look at what make dev was doing, just quickly added the script entry.

Comment (aaronmgdr) [reply package.json:8]: i think this makes sense

**Pull request merged**